server/routes/teams.ts

```diff
@@ -65,16 +65,22 @@ export default function routes({
       const channelRecommendations = monitoringChannelService.generateChannelRecommendations(team)
       const channelTree = monitoringChannelService.generateChannelTree(channelRecommendations)
 
-      res.render('pages/teamOverview', {
-        teamName: team.name,
-        formatTeamNameURL: encodeURIComponent(team.name),
-        teamSlug: team.slug,
-        teamAlertSummary,
+      const slackTeamId = 'T02DYEB3A'
+
+      const displayTeamOverview = {
```

Comment:
The const wants naming the same as the name of the attribute you're passing into the template. Is it a team or is it an overview?

Reply:
Nothing wrong with the object per se so happy as is, just the naming of the variable and the attribute you pass in so it is clearer 🙂